opsdroid/connector/slack/create_events.py

```diff
@@ -257,3 +258,15 @@ async def view_closed_triggered(self, event, channel):
             target=event["user"]["id"],
             connector=self.connector,
         )
+
+    async def slash_command(self, event, channel):
+        """Send a Slash command event"""
+
+        command = slack_events.SlashCommand(
+            event,
+            user=event["user_id"],
+            target=event["channel_id"],
+            connector=self.connector,
+        )
+        command.update_entity("command", event["command"])
```

Comment:
Commands can take arguments right? How do these filter through into the event?

Reply:
that's correct. The first argument that we pass to  SlashCommand (InteractiveAction) is the payload.
The payload contains a `text` key which will have the arguments of the command.
I can be more explicit in Line 266, and add `payload=event` instead of just `event`.

Also I think that it will be useful to add this to the documentation.

Another option that I can think of is to create an additional attribute to SlashCommands, ie: `arguments` and pass the `event["text"]` as a string or a list of strings to the `arguments` attribute.

Reply:
that's correct. The first argument that we pass to  SlashCommand (InteractiveAction) is the payload.
The payload contains a `text` key which will have the arguments of the command. So if we want to retrieve the arguments from a skill, we would have to call `self.payload["text"]`

I can be more explicit in Line 266, and add `payload=event` instead of just `event`.

Also I think that it will be useful to add this to the documentation.

Another option that I can think of is to create an additional attribute to SlashCommands, ie: `arguments` and pass the `event["text"]` as a string or a list of strings to the `arguments` attribute.

Reply:
I think adding the `event[“text”]` option is the best idea. And making that part of the documentation would be easy parity with Slack.

Reply:
Can you be more explicit, please? I have written  `event["text"]` two times in my last comment, and I am not quite sure which one are you referring to. :)

Reply:
I am going to leave this as it is, and fix the docummentation.